client/my-sites/stats/features/modules/stats-utm/stats-module-utm.jsx

```diff
@@ -1,13 +1,16 @@
+import config from '@automattic/calypso-config';
 import page from '@automattic/calypso-router';
 import { StatsCard } from '@automattic/components';
 import { localizeUrl } from '@automattic/i18n-utils';
+import { memo } from '@wordpress/element';
```

Comment:
Let's import from `react` to be consistent across other imports.

Reply:
This change was left over from debugging and has been removed